starcounter-include.html

```diff
@@ -245,10 +250,6 @@
             if(partial){
                 if(partial.Html !== undefined){
                     return partial.Html;
-                } else if (partial.LauncherLayoutInfo) {
-                    //workaround for https://github.com/StarcounterPrefabs/Launcher/issues/224
-                    //should be removed when the original issue with not patched Html is resolved
-                    return partial.LauncherLayoutInfo.MergedHtml;
```

Comment:
The issue seems fixed. However since, CompositionProvider **is** providing this `MergedHtml` to stick to the rule, that server is the source of truth, and to reduce operations, we should probably still use it.
https://github.com/StarcounterApps/CompositionProvider/blob/develop/src/CompositionProvider/Program.cs#L50-L74

WDYT?